### PR TITLE
Rename file numbers

### LINKED ISSUE #69: Fix the off-by-1 error between the section list and file names
Shouldn't have done 0-based index lol

### PR DESCRIPTION
**Pull request by JasonFreeberg**
Fixes the disconnect between the TOC numbers and the numbers in the file names. 

Also, this adds section numbers to all files:

- H1 headers get a single number (1, 2, 3, etc.)
- H2 headers get a decimal (1.1, 1.,2, 1.3, etc.)
- H3 headers get another decimal (1.1.1, 1.1.2, 1.1.3, etc.)

Fixes #69

### CODE REVIEW

File: instructions/4-create-postgres-on-azure.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ It should report:
 ```
 <path>/setup/postgresql.jar: Java archive data (JAR)
 ```
+
 ### Create Application Settings
```
Comment (jamesfalkner) [inline instructions/4-create-postgres-on-azure.md:109]: Does this need a decimal instead of `###` ?

**Comment by jamesfalkner** [comment]
@JasonFreeberg I left one very minor comment above, but was going to just merge it until I noticed there are conflicts! Can you resolve these? If you do, just go ahead and commit it as I've reviewed the changes.

**Review by danieloh30 — APPROVED** [review]
Looks good to me except for James' comment :)

File: instructions/4-create-postgres-on-azure.md (hunk 1 above)
Comment (JasonFreeberg) [reply instructions/4-create-postgres-on-azure.md:109]: Good catch! Yeah will fix that

**Comment by JasonFreeberg** [comment]
@jamesfalkner @danieloh30 -- Sounds good, will resolve and merge

**Review by danieloh30 — APPROVED** [review]

**Pull request merged**